blazor/dropdown-list/value-binding.md

```diff
@@ -0,0 +1,216 @@
+---
+layout: post
+title: Value Binding in Blazor DropDown List Component | Syncfusion
+description: Checkout and learn here all about Value Binding in Syncfusion Blazor DropDown List component and more.
+platform: Blazor
+control: DropDown List
+documentation: ug
+---
+
+# Value Binding
+
+The value binding as the process of passing values between a component and its parent. There are two methods for binding values, as shown below.
+
+    * bind-Value Binding 
+    * bind-Index Binding
+
+## Bind Value Binding
+Value binding can be achieved by using the [@bind-Value](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Value) attribute and it supports string, int, Enum, bool and complex types. If component value has been changed, it will affect the all places where you bind the variable for the bind-value attribute.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/value-binding/bind-value.razor %}
+
+{% endhighlight %}
+
+![Blazor DropDownList with Bind Value](./images/value-binding/blazor_dropdown_bind-value.png)
+
+## Index Value Binding
+
+Index value binding can be achieved by using [@bind-Index](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Index) attribute and it supports int and int nullable types. By using this attribute you can bind the values respective to its index.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/value-binding/index-value.razor %}
+
+{% endhighlight %}
+
+![Blazor DropDownList with Bind Value](./images/value-binding/blazor_dropdown_index-value.png)
+
+## Text and Value
+
+[Value](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.FieldSettingsModel.html#Syncfusion_Blazor_DropDowns_FieldSettingsModel_Value) field mapped to the component maintains the unique value of the item in the data source. The [Text](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.FieldSettingsModel.html#Syncfusion_Blazor_DropDowns_FieldSettingsModel_Text) field used to display the text in the popup list items for the respective value. We can specify these properties within the `DropDownListFieldSettings` tag helper.

```

Comment:
Use this - The DropdownList [DropDownListFieldSettings.Value](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.FieldSettingsModel.html#Syncfusion_Blazor_DropDowns_FieldSettingsModel_Value) and [DropDownListFieldSettings.Text](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.FieldSettingsModel.html#Syncfusion_Blazor_DropDowns_FieldSettingsModel_Text) properties point to the corresponding names of the model. The `Value` field mapped to the component maintains the unique value of the item in the data source, and the `Text` field is mapped to display the text in the popup list items for the respective text value.